src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -300,16 +300,12 @@ extern EntityIdVector subToEntityIdVector(const orion::BSONObj& sub);
 *
 * subToNotifyList -
 */
-void subToNotifyList
+void  subToNotifyList
```

Comment:
Lint

```suggestion
void  subToNotifyList
```

Reply:
Fixed in [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475)